### PR TITLE
Add weeklyReportFive

### PR DESCRIPTION
**Pull request by robin-debry**

### CODE REVIEW

**Review by EnzoGuillouche — APPROVED** [review]
Everything's alright. Merging approved.

File: Documents/Management/WeeklyReports/mainWeeklyReport.md [hunk 1]
```diff
@@ -230,6 +234,41 @@ The challenges encountered this week are still the same as the previous week. Th
...
+   - Finish the test plan on Wednesday.
+   - Finish the user manual on Wednesday.
+
+## Challenges of the fifth week
+
+The challenges encountered this week are still the same as the previous week but the team managed to finish the work with LEDs.
+
+## Plan for the sixth week
+
+- Prepare the final presentation
```
Comment (MathiasGagnepain) [inline Documents/Management/WeeklyReports/mainWeeklyReport.md:266]: A dot is missing.

File: Documents/Management/WeeklyReports/mainWeeklyReport.md [hunk 2]
```diff
@@ -230,6 +234,41 @@ The challenges encountered this week are still the same as the previous week. Th
...
+
+## Conclusion of the fifth week
+
+The fifth week was short, the team finished simulating the functionality of each component with LEDs. The team also managed to finish the documentation like the test plan and the User manual. The team is still motivated to finish the project with the presentation next week.
```
Comment (MathiasGagnepain) [inline Documents/Management/WeeklyReports/mainWeeklyReport.md:270]: why there is a U in uppercase in the middle of the sentence?

File: Documents/Management/WeeklyReports/mainWeeklyReport.md [hunk 3]
```diff
@@ -230,6 +234,41 @@ The challenges encountered this week are still the same as the previous week. Th
...
 
+## Week 5 (8-10 April 2024)
+
+[week 5](weeklyReportFifthWeek.md)
+
+The fifth week was a shorter week due to the Hackathon. The team finished working with LEDs. The team also finished working on the test plan, the user manual and the code documentation.
+
+## Topics of the fifth week
+
+1. Finish to Manage each component with LEDs.
```
Comment (MathiasGagnepain) [inline Documents/Management/WeeklyReports/mainWeeklyReport.md:245]: Why there is M in uppercase in the middle of the sentence?

File: Documents/Management/WeeklyReports/weeklyReportFifthWeek.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+[Cumulative Weekly Reports](mainWeeklyReport.md)
+
+## Week 5 (8-10 April 2024)
+
+The fifth week was a shorter week due to the Hackathon. The team finished working with LEDs. The team also finished working on the test plan, the user manual, and the code documentation.
+
+## This week's topics
+
+1. Finish to Manage each component with LEDs.
```
Comment (MathiasGagnepain) [inline Documents/Management/WeeklyReports/weeklyReportFifthWeek.md:11]: Same as in the previous document, why there is a M in uppercase in the middle of the sentence?

File: Documents/Management/WeeklyReports/weeklyReportFifthWeek.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+   - Finish the test plan on Wednesday.
+   - Finish the user manual on Wednesday.
+
+## Challenges
+
+The challenges encountered this week are still the same as the previous week but the team managed to finish the work with LEDs.
+
+## Next week's plan
+
+- Prepare the final presentation
```
Comment (MathiasGagnepain) [inline Documents/Management/WeeklyReports/weeklyReportFifthWeek.md:32]: Same as in the previous document, you forgor a dot.

File: Documents/Management/WeeklyReports/weeklyReportFifthWeek.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+## Conclusion
+
+The fifth week was short, the team finished simulating the functionality of each component with LEDs. The team also managed to finish the documentation like the test plan and the User manual. The team is still motivated to finish the project with the presentation next week.
```
Comment (MathiasGagnepain) [inline Documents/Management/WeeklyReports/weeklyReportFifthWeek.md:36]: Same as in the previous document, why there is a U in uppercase in the middle of the sentence?

**Review by MathiasGagnepain — CHANGES_REQUESTED** [review]
Minor issues.

File: Documents/Management/WeeklyReports/weeklyReportFifthWeek.md (hunk 4 above)
Comment (MathiasGagnepain) [reply Documents/Management/WeeklyReports/weeklyReportFifthWeek.md]: fixed

File: Documents/Management/WeeklyReports/weeklyReportFifthWeek.md (hunk 5 above)
Comment (MathiasGagnepain) [reply Documents/Management/WeeklyReports/weeklyReportFifthWeek.md]: fixed

File: Documents/Management/WeeklyReports/mainWeeklyReport.md (hunk 1 above)
Comment (MathiasGagnepain) [reply Documents/Management/WeeklyReports/mainWeeklyReport.md]: fixed

File: Documents/Management/WeeklyReports/mainWeeklyReport.md (hunk 2 above)
Comment (MathiasGagnepain) [reply Documents/Management/WeeklyReports/mainWeeklyReport.md]: fixed

File: Documents/Management/WeeklyReports/weeklyReportFifthWeek.md (hunk 6 above)
Comment (MathiasGagnepain) [reply Documents/Management/WeeklyReports/weeklyReportFifthWeek.md]: fixed

**Review by MathiasGagnepain — APPROVED** [review]
Perfect.

**Pull request merged**